### ISSUE TITLE
Database Microsoft with multiple Schema

### ISSUE THREAD

**Issue by francescobianco** [Myddleware/myddleware#523]
In situation with large Microsoft SQL Database many admin looking for using different database schema instead of the default one called 'dbo'

![Screenshot from 2021-04-29 16-29-02](https://user-images.githubusercontent.com/472171/116568426-c3846b80-a908-11eb-88ad-dbc9d22c9ac8.png)

We looking for the possibility to choose the database SCHEMA like this example
We actually manage existing schema with a tool called Adminer, it has a similar select box

![Screenshot from 2021-04-29 16-36-50](https://user-images.githubusercontent.com/472171/116569097-5d4c1880-a909-11eb-88f8-326217e2cafd.png)

This has many implication in all queries 

For example this CREATE query is wrong because it refer to DBO schema:

```
INSERT INTO [People] ([Name], [Surname]) VALUES ('Frank', 'White')
```

The right one must be something like this

```
INSERT INTO [CUSTOM_SCHEMA].[People] ([Name], [Surname]) VALUES ('Frank', 'White')
```

**Comment by Myddleware** [comment]
Hi @francescobianco ,
Yes I understand your need. Could you test a workaround ? Could you try to add the shema inside the database field in the connector like shema.database ?
Does it work ?

**Comment by francescobianco** [comment]
Yes, of course, we have found a work-around based on this four methods

- Customizing the buildQuery(..) to pass the schema name into FROM statement
- Customizing the create(..) to pass the modified module name prefixed by schema
- Customizing the udpate(..) to pass the modified module name prefixed by schema 
- Customizing the read_last(..) to pass the modified module name prefixed by schema

It works in this way, it just and idea for future implementation, not blocking for now

**Comment by Myddleware** [comment]
Thanks for your feedback. 
I have added the flag "enhancement" to this issue. 
Have a nice day.
